addons/event/models/event_mail.py

```diff
@@ -297,10 +300,16 @@ def execute(self):
             email_values = {
                 'author_id': author.id,
             }
-            if not reg_mail.scheduler_id.template_ref.email_from:
+            try:
+                template = reg_mail.scheduler_id.template_ref
+            except MissingError:
+                _logger.warning("Cannot process ticket %s, because Mail Scheduler %s has reference to non-existent template", reg_mail.registration_id, reg_mail.scheduler_id)
```

Comment:
Might be good to have a final log in the chatter to specify some mails couldn't be sent, just so people are aware some were not sent.

Reply:
I understand that current solution is a bit obscure, but I don't see a better patch for stable.
If we post log in event chatter, the message will be repeated every hour, right? It doesn't look good for me. If you have other ideas let me know

Admin should check Mail Scheduler menu to check if everything is ok.
It can be improved, but the specification should be reviewed by PO fist and the improvements should be done in master. 

![image](https://user-images.githubusercontent.com/186131/235121270-4e1de17b-75c7-4031-bf38-e510c9279f4f.png)

Reply:
I think it's rare enough that it wouldn't be a huge issue but you're right it's not very tidy either...

As you said, let's fix the crash here and come up with something nice on master later :)